topics/latency.md

```diff
@@ -159,19 +159,19 @@ Only do them if you require them, and if you are familiar with them.
 Single threaded nature of Valkey
 -------------------------------
 
-Valkey uses a *mostly* single threaded design. This means that a single process
-serves all the client requests, using a technique called **multiplexing**.
-This means that Valkey can serve a single request in every given moment, so
-all the requests are served sequentially. This is very similar to how Node.js
+Valkey uses a *mostly* single threaded design for command execution. This means that a single process
+executes all the client commands sequentially, using a technique called **multiplexing**.
+While multiple commands can have their I/O operations processed concurrently in the background,
+only one command can be executed at any given moment. This is similar to how Node.js
 works as well. However, both products are not often perceived as being slow.
-This is caused in part by the small amount of time to complete a single request,
+This is caused in part by the small amount of time to complete a single command execution,
 but primarily because these products are designed to not block on system calls,
 such as reading data from or writing data to a socket.
 
-I said that Valkey is *mostly* single threaded since
-we use threads in Valkey in order to perform some slow I/O operations in the
-background, mainly related to disk I/O, but this does not change the fact
-that Valkey serves all the requests using a single thread.
+I said that Valkey is *mostly* single threaded.
+We do use threads in Valkey in order to perform I/O operations (including command parsing and I/O operations)
+in the background, but this does not change the fact
+that Valkey executes all the commands sequentially using a single thread.
```

Comment:
```suggestion
```
I think the multi-threaded is fully addressed earlier, do we need to repeat it here?

Reply:
Agree